apps/vite/src/components/AddEventModal.tsx

```
@@ -10,19 +10,20 @@ import colors from "../constants/colors"
-import { toast } from "react-hot-toast"
-import { apiBaseUrl, createEvent, getCalendarList } from "../helper/api"
+import { CalendarEvent } from "@miti/types"
+import { apiBaseUrl } from "../helper/api"
 import DropDown from "./DropDown"
 import Spinner from "./Spinner"
+import { useCalendarList } from "@miti/query/calendar"
+import { useCreateEvent } from "@miti/query/event"
 
 function getCombinedDateTime(date: Date, time: string) {
   const timeParts = time.split(":")
   date.setHours(parseInt(timeParts[0] ?? "", 10))
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)